FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -350,20 +363,21 @@ public void ExecuteServiceAccountLogin()
         }
 
         [Test]
+        [Category("general")]
         public void ExecuteInformationSchema()
         {
-            using var conn = new FireboltConnection(SYSTEM_CONNECTION_STRING);
+            using var conn = new FireboltConnection(USER_CONNECTION_STRING);
             conn.Open();
             DbCommand command = conn.CreateCommand();
-            command.CommandText = "SELECT table_name, number_of_rows from information_schema.tables";
```

Comment:
It looks like but it is  not. The idea here was to create a sanity test of `FireboltDataReader`. This is the reason to run query that returns columns of different typs: string and integer. In the beginning it was `table_name` and `number_of_rows` and this test worked well for moths. But suddenly `number_of_rows` sometimes became `null`, so I just changed it to `length(table_name)` to be sure that `int` value is returned. 

Probably we should just rename this test to avoid confusion?

Reply:
Yeah, let's rename so the test name reflects what we're testing instead of how. 
I think it might be redundant since we're doing fetches elsewhere, but I'd rather have redundant tests than test gaps.

Reply:
renamed